### PR TITLE
Add update options into a couple places for channel

### PR DESCRIPTION
**Pull request by ferhatelmas**

### CODE REVIEW

File: src/channel.ts [hunk 1]
```diff
@@ -541,15 +548,18 @@ export class Channel<
    *
    * @param {{channel_role: Role, user_id: string}[]} roles List of role assignments
    * @param {Message<AttachmentType, MessageType, UserType>} [message] Optional message object for channel members notification
+   * @param {ChannelUpdateOptions} [options] Option object, configuration to control the behavior while updating
    * @return {Promise<UpdateChannelAPIResponse<AttachmentType, ChannelType, CommandType, MessageType, ReactionType, UserType>>} The server response
    */
   async assignRoles(
     roles: { channel_role: Role; user_id: string }[],
     message?: Message<AttachmentType, MessageType, UserType>,
+    options?: ChannelUpdateOptions,
```
Comment (mahboubii) [inline src/channel.ts:557]:
```suggestion
    options?: ChannelUpdateOptions = {},
```

File: src/channel.ts [hunk 2]
```diff
@@ -558,15 +568,18 @@ export class Channel<
    *
    * @param {{user_id: string, channel_role?: Role}[]} members An array of members to invite to the channel
    * @param {Message<AttachmentType, MessageType, UserType>} [message] Optional message object for channel members notification
+   * @param {ChannelUpdateOptions} [options] Option object, configuration to control the behavior while updating
    * @return {Promise<UpdateChannelAPIResponse<AttachmentType, ChannelType, CommandType, MessageType, ReactionType, UserType>>} The server response
    */
   async inviteMembers(
     members: { user_id: string; channel_role?: Role }[] | string[],
     message?: Message<AttachmentType, MessageType, UserType>,
+    options?: ChannelUpdateOptions,
```
Comment (mahboubii) [inline src/channel.ts:577]:
```suggestion
    options?: ChannelUpdateOptions = {},
```

File: src/channel.ts [hunk 3]
```diff
@@ -575,12 +588,18 @@ export class Channel<
...
    * @param {string[]} members An array of member identifiers
    * @param {Message<AttachmentType, MessageType, UserType>} [message] Optional message object for channel members notification
+   * @param {ChannelUpdateOptions} [options] Option object, configuration to control the behavior while updating
    * @return {Promise<UpdateChannelAPIResponse<AttachmentType, ChannelType, CommandType, MessageType, ReactionType, UserType>>} The server response
    */
-  async removeMembers(members: string[], message?: Message<AttachmentType, MessageType, UserType>) {
+  async removeMembers(
+    members: string[],
+    message?: Message<AttachmentType, MessageType, UserType>,
+    options?: ChannelUpdateOptions,
```
Comment (mahboubii) [inline src/channel.ts:597]:
```suggestion
    options?: ChannelUpdateOptions = {},
```

File: src/channel.ts [hunk 4]
```diff
@@ -589,12 +608,18 @@ export class Channel<
...
    * @param {string[]} members An array of member identifiers
    * @param {Message<AttachmentType, MessageType, UserType>} [message] Optional message object for channel members notification
+   * @param {ChannelUpdateOptions} [options] Option object, configuration to control the behavior while updating
    * @return {Promise<UpdateChannelAPIResponse<AttachmentType, ChannelType, CommandType, MessageType, ReactionType, UserType>>} The server response
    */
-  async demoteModerators(members: string[], message?: Message<AttachmentType, MessageType, UserType>) {
+  async demoteModerators(
+    members: string[],
+    message?: Message<AttachmentType, MessageType, UserType>,
+    options?: ChannelUpdateOptions,
```
Comment (mahboubii) [inline src/channel.ts:617]:
```suggestion
    options?: ChannelUpdateOptions = {},
```

File: src/channel.ts [hunk 5]
```diff
@@ -527,12 +528,18 @@ export class Channel<
...
    * @param {string[]} members An array of member identifiers
    * @param {Message<AttachmentType, MessageType, UserType>} [message] Optional message object for channel members notification
+   * @param {ChannelUpdateOptions} [options] Option object, configuration to control the behavior while updating
    * @return {Promise<UpdateChannelAPIResponse<AttachmentType, ChannelType, CommandType, MessageType, ReactionType, UserType>>} The server response
    */
-  async addModerators(members: string[], message?: Message<AttachmentType, MessageType, UserType>) {
+  async addModerators(
+    members: string[],
+    message?: Message<AttachmentType, MessageType, UserType>,
+    options?: ChannelUpdateOptions,
```
[the remark this answers is not in the document]
Comment (ferhatelmas) [reply src/channel.ts:537]: true, forgot it

**Review by mahboubii — DISMISSED** [review]

**Review by mahboubii — APPROVED** [review]

**Pull request merged**